docker/README.md

```diff
@@ -68,41 +68,38 @@ docker rm -f mms
 
 ## Available pre-built continers
 We have 5 containers tags available on [Docker Hub](https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/).
```

Comment:
```suggestion
We have 8 containers tags available on [Docker Hub](https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/).
```

Reply:
```suggestion
We have following container tags available on [Docker Hub](https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/).
```
I propose to remove number of tags available so that we won't have to update this file every release but also so that we won't have discrepancy in number we announce and actual available containers.